tests/topotests/bgp_multiview_topo1/test_bgp_multiview_topo1.py

```diff
@@ -152,7 +152,6 @@ def setup_module(module):
 
     # Starting PE Hosts and init ExaBGP on each of them
     print('*** Starting BGP on all 8 Peers in 10s')
-    sleep(10)
```

Comment:
Log output in the preceding line is wrong now?

Reply:
fixed